clients/js/test/plugins/collection/updateDelegate.test.ts

```diff
@@ -919,3 +933,147 @@ test('it can update an owner-managed plugin on an asset as collection update add
     },
   });
 });
+
+// Works
+test('it can add asset to collection as collection update delegate', async (t) => {
+  const umi = await createUmi();
+  const assetOwner = umi.identity;
+  const collectionUmi = await createUmi();
+
+  const assetSigner = generateSigner(umi);
+  const assetAddress = assetSigner.publicKey;
+  await createV2(umi, {
+    asset: assetSigner,
+    name: 'My Asset',
+    uri: 'https://example.com/my-asset.json',
+  }).sendAndConfirm(umi);
+
+  const collectionSigner = generateSigner(collectionUmi);
+  const collectionAddress = collectionSigner.publicKey;
+  await createCollection(collectionUmi, {
+    collection: collectionSigner,
+    name: 'My Collection',
+    uri: 'https://example.com/my-collection.json',
+  });
+
+  await addCollectionPlugin(collectionUmi, {
+    collection: collectionAddress,
+    plugin: {
+      type: 'UpdateDelegate',
+      authority: { type: 'Address', address: assetOwner.publicKey },
+      additionalDelegates: [],
+    },
+  }).sendAndConfirm(collectionUmi);
+
+  await updateV2(umi, {
+    asset: assetAddress,
+    newCollection: collectionAddress,
+    newUpdateAuthority: updateAuthority('Collection', [collectionAddress]),
+  }).sendAndConfirm(umi);
```

Comment:
Isn't this using ui as the authority for the call but should be using the update delegate?  Which I think for clarity should not be set to the owner?

Reply:
Ah you are correct.